demo/site/src/app/[lang]/layout.tsx

```diff
@@ -0,0 +1,21 @@
+import { readFile } from "fs/promises";
+
+import { IntlProvider } from "./IntlProvider";
+
+const messagesCache: Record<string, unknown> = {};
+async function loadMessages(lang: string) {
+    if (messagesCache[lang]) return messagesCache[lang];
+    const path = `./lang-compiled/${lang}.json`;
+    const messages = JSON.parse(await readFile(path, "utf8"));
```

Comment:
Previously we did this with a dynamic import. Is reading the file directly better?

Reply:
I was not able to get that working, tried:
- ``const path = `@src/../lang-compiled/${lang}.json`;``
- ``const path = `../../../../lang-compiled/${lang}.json`;``
- ``const path = `${process.cwd()}/lang-compiled/${lang}.json`;``